modules/account-lib/src/coin/avaxc/transferBuilder.ts

```diff
@@ -0,0 +1,22 @@
+import { TransferBuilder as EthTransferBuilder } from '../eth';
+
+/** AVAXC transfer builder */
+export class TransferBuilder extends EthTransferBuilder {
+  /**
+   * Get the prefix used in generating an operation hash for sending tokens
+   *
+   * @returns the string prefix
+   */
+  protected getTokenOperationHashPrefix(): string {
+    return 'AVAXC-ERC20';
```

Comment:
@CapnMigraine 
This string should match the one on `SendMultisigToken` method on the contract (which is `AVAX-ERC20` at the moment).
Should we update the contract PR or this PR instead?

Reply:
@shiksb I think we can assume that if we were to support an asset like `AVAX`, but with multiple eth-like chains, the asset should be the same across them —i.e. `AVAX` on the X/P chains—, and the contract would be the same too. We can use `AVAX-ERC20` and `AVAX`.